### ISSUE TITLE
Sharing screen shows different content than other platforms

### ISSUE THREAD

**Issue by mpivchev** [nextcloud/android#13223]
Old content does not appear on the Activity tab:

iOS:
<img src="https://github.com/nextcloud/android/assets/6960329/95f11cce-25a0-4b9f-93bc-2901ef44abe1" width="300" />

Web:
<img src="https://github.com/nextcloud/android/assets/6960329/90b5fb8c-b08b-476c-b8c1-88fb5e484093" width="300" />

Android:

<img src="https://github.com/nextcloud/android/assets/6960329/d8f41292-7a2e-4812-8513-f18f1e9e7913" width="300" />
<img src="https://github.com/nextcloud/android/assets/6960329/90877b24-1b45-41b9-9041-ea21cb81e7f3" width="300" />

Issues:
- On Android, comments are not shown
- Activity dates don't show the year, so it's confusing when scrolling up goes from December to February without any year to indicate.
- Android shows "New version was created" but on iOS and Web this is not present (Web has its own tab for this). Different content is shown on different platforms. (@jancborchardt)

**Comment by jancborchardt** [comment]
Good point @mpivchev it seems that only iOS does it correctly – with activity + comments, chronologically correctly interleaved.

Also cc @sorbaugh for the web interface.
